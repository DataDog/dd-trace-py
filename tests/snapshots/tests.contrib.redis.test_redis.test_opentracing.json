[[
  {
    "name": "redis_get",
    "service": "redis_svc",
    "resource": "redis_get",
    "trace_id": 0,
    "span_id": 1,
    "parent_id": 0,
    "type": "",
    "error": 0,
    "meta": {
      "_dd.p.dm": "-0",
      "language": "python",
      "runtime-id": "03c105104b1e4097a4b4e5dad57fcb39"
    },
    "metrics": {
      "_dd.agent_psr": 1.0,
      "_dd.top_level": 1,
      "_dd.tracer_kr": 1.0,
      "_sampling_priority_v1": 1,
      "process_id": 22962
    },
    "duration": 557380,
    "start": 1691164288663325952
  },
     {
       "name": "redis.command",
       "service": "redis",
       "resource": "GET cheese",
       "trace_id": 0,
       "span_id": 2,
       "parent_id": 1,
       "type": "redis",
       "error": 0,
       "meta": {
         "_dd.p.dm": "-0",
         "component": "redis",
         "db.system": "redis",
         "out.host": "localhost",
         "redis.raw_command": "GET cheese",
         "span.kind": "client"
       },
       "metrics": {
         "_dd.measured": 1,
         "_dd.top_level": 1,
         "network.destination.port": 6379,
         "out.redis_db": 0,
         "redis.args_length": 2
       },
       "duration": 373714,
       "start": 1691164288663468873
     }]]
