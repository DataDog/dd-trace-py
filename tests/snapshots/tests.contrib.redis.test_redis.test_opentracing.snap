[[{"name" "redis_get"
   "service" "redis_svc"
   "resource" "redis_get"
   "error" 0
   "span_id" 0
   "trace_id" 0
   "parent_id" nil
   "start" 1596062188665713920
   "duration" 3061080
   "meta" {"runtime-id" "601815e8709646a9ac897c59376c80b6"}
   "metrics" {"_dd.agent_psr" 1.0
              "_dd.tracer_kr" 1.0
              "_sampling_priority_v1" 1
              "system.pid" 76411}}
      {"name" "redis.command"
       "service" "redis"
       "resource" "GET cheese"
       "type" "redis"
       "error" 0
       "span_id" 1
       "trace_id" 0
       "parent_id" 0
       "start" 1596062188666033000
       "duration" 2694000
       "meta" {"redis.raw_command" "GET cheese"
               "out.host" "localhost"}
       "metrics" {"_dd.measured" 1
                  "out.port" 6379
                  "out.redis_db" 0
                  "redis.args_length" 2}}]]
